osu.Game/Tests/Visual/Multiplayer/TestMultiplayerClient.cs

```diff
@@ -307,15 +386,15 @@ public override async Task SendMatchRequest(MatchUserRequest request)
             }
         }
 
-        public override Task StartMatch()
+        public override async Task StartMatch()
         {
             Debug.Assert(Room != null);
 
             ChangeRoomState(MultiplayerRoomState.WaitingForLoad);
             foreach (var user in Room.Users.Where(u => u.State == MultiplayerUserState.Ready))
                 ChangeUserState(user.UserID, MultiplayerUserState.WaitingForLoad);
 
-            return ((IMultiplayerClient)this).LoadRequested();
+            await ((IMultiplayerClient)this).LoadRequested();
```

Comment:
Was this change required for a specific reason? If so, should be commented; if not, should be reverted.

Reply:
Reverted.